### PR TITLE
Remove 2.2 from AC table, fix Airflow version in runtime table

### PR DESCRIPTION
**Pull request by jedcunningham**

### CODE REVIEW

File: software/ac-support-policy.md [hunk 1]
```diff
@@ -128,7 +128,6 @@ The following tables contain the exact lifecycle for each published version of A
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
 | [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
 | [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
-| [2.2](https://github.com/astronomer/ap-airflow/blob/master/2.2.0/CHANGELOG.md)       | Oct 12, 2021   | April 2022              |
 | [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
```
Comment (jedcunningham) [inline software/ac-support-policy.md:131]: We've hit a point where all of our releases are LTS... Seems a little odd to duplicate the table for stable also?

Comment (jwitz) [reply software/ac-support-policy.md:131]: Good point, but I'm guessing a purely stable release is coming up soon which will need to be added to only the first table? cc @paolaperaza

Comment (jedcunningham) [reply software/ac-support-policy.md:131]: No, 2.3 is the last AC release.

File: astro/runtime-version-lifecycle-policy.md [hunk 2]
```diff
@@ -80,8 +80,8 @@ The following table contains the exact lifecycle for each published version of A
 | Runtime Version                                     | Apache Airflow version | Release Date      | End of Maintenance Date |
 | --------------------------------------------------- | ---------------------- | ----------------- | ----------------------- |
 | [4.1.x](runtime-release-notes.md#astro-runtime-410) | 2.2.4                  | February 22, 2022 | August 2022             |
-| [4.2.x](runtime-release-notes.md#astro-runtime-420) | 2.2.4                  | March 10, 2022    | September 2022          |
-| [5.0.x](runtime-release-notes.md#astro-runtime-500) | 2.3.0                  | April 30, 2022    | October 2022            |
+| [4.2.x](runtime-release-notes.md#astro-runtime-420) | 2.2.4-2.2.5            | March 10, 2022    | September 2022          |
+| [5.0.x](runtime-release-notes.md#astro-runtime-500) | 2.3.0-2.3.3            | April 30, 2022    | October 2022            |
```
Comment (jwitz) [inline astro/runtime-version-lifecycle-policy.md:85]:
```suggestion

:::info
Each Runtime version in a given minor series supports only a single version of Apache Airflow. For specific version compatibility information, see [Runtime release notes](runtime-release-notes.md).
:::

```
I think we need some sort of note here to indicate that you can't run all versions of Airflow in the range specified in the table. WDYT?

**Review by jwitz — COMMENTED** [review]
One minor comment, otherwise LGTM

**Review by jwitz — APPROVED** [review]

**Pull request merged**